config/printer-flashforge-creator-pro-2018.cfg

```diff
@@ -0,0 +1,175 @@
+# This file contains common pin mappings for the FlashForge-Creator-Pro
+# To use this config, the firmware should be compiled for
+# the Atmel atmega2560.
+# Use 57600 baud for the bootloader
```

Comment:
Thanks.  In general this looks good to me.  One comment though - can you elaborate on what `Use 57600 baud for the bootloader`.  I'm leery of mentioning "baud" without clarification, because for some strange reason people tend to set the Klipper baud rates to incorrect values.

-Kevin

Reply:
Yes, I think that the best way would be to an an optional BAUDRATE parameter in the makefile and specify the command in the config file.
I will edit the baud comment and create a PR for the makefile.

-Marc-André

Reply:
As a general rule, we don't add flash rules to the build code (there are just too many different ways to flash various boards).  If you have a specific flash command, I recommend you add the full command to the comments of the config file - for example, see: https://github.com/KevinOConnor/klipper/blob/master/config/generic-melzi.cfg

-Kevin